plugins/lcvcore/src/qmatop.h

```diff
@@ -20,7 +21,16 @@ class QMatOp : public QObject{
 
     QMat* nullMat() const;
 
+    static void appendItem(lv::QmlObjectList* list, QObject* item);
+    static int itemCount(lv::QmlObjectList* list);
+    static QObject* itemAt(lv::QmlObjectList* list, int idx);
+    static void removeItemAt(lv::QmlObjectList* list, int idx);
+    static void clearItems(lv::QmlObjectList* list);
```

Comment:
I would wrap these functions into a separate nested class, to separate MatOp functionality vs functionality on lists.

```cpp
public:

   class ListOperations{
       static void appendItem(lv::QmlObjectList* list, QObject* item);
       // ...
   }